util/decoding/context.go

```diff
@@ -0,0 +1,28 @@
+package decoding
+
+import "context"
+
+type requestContextKey string
+
+const (
+	reqCtxKeyMaxLen     = requestContextKey("server-decoding-plugin-context-max-length")
+	reqCtxKeyGzipMaxLen = requestContextKey("server-decoding-plugin-context-gzip-max-length")
+)
+
+func AddServerDecodingBodyMaxLen(ctx context.Context, maxLen int64) context.Context {
```

Comment:
Is this used?

Reply:
Not used currently, but included for completeness. I'm open to removing it entirely, because we'd only need to annotate that context key if we *really* change up how OPA handles requests.

Reply:
We can remove this and add later if needed. Same for the getter as well. We can remove for now.

Reply:
Just saw this bit of feedback. I'll prune the extra method and push up the changes.